src/AutoRest.CSharp/LowLevel/Output/LowLevelRestClient.cs

```diff
@@ -80,18 +80,25 @@ private IEnumerable<LowLevelRestClientMethod> BuildAllMethods()
                         schemaDocumentation = GetSchemaDocumentationsForParameter(bodyParameter);
                     }
 
+                    // Inject the RequestOptions
+                    CSharpType requestType = new CSharpType (typeof(Azure.RequestOptions)).WithNullable(true);
```

Comment:
My gut feel is to handle this parameter in the writer like we did with the cancellation token before.

Reply:
I could, but I though pushing up params to the model was preferable?